src/Fantomas.Tests/LetBindingTests.fs

```diff
@@ -1557,3 +1557,43 @@ let myFun (a: decimal) b c : decimal = a + b + c
 let expensiveToCompute : int = 0
 let myFun (a: decimal) b c : decimal = a + b + c
 """
+
+[<Test>]
```

Comment:
Could you move this test to https://github.com/fsprojects/fantomas/blob/master/src/Fantomas.Tests/DotGetTests.fs please?

Reply:
Absolutely. Done :) rebased